signac/contrib/schema.py

```diff
@@ -36,6 +42,8 @@ def _collect_by_type(values):
         A mapping of types to a set of input values of that type.
 
     """
+    # TODO: This function should be moved to project.py in version 2.0.
```

Comment:
I find it valuable to include TODO so we can search for these comments and clean them up in version 2.0. If we remove the word, it will be hard to find comments like this one.

Reply:
Yeah I personally still prefer to use TODOs. I would rather have unactionable items show up for a while rather than not have the TODO and therefore forget about the task when it does become actionable.